extensions/wikia/CommunityPage/styles/components/_admins-module.scss

```diff
@@ -45,17 +45,24 @@ $admins-module-item-width: 80px;
 	.community-page-admin-item {
 		margin: auto;
 		max-width: 50%;
-
-		svg {
-			fill: $color-links;
-			height: 18px;
-			width: 18px;
-		}
+		position: relative;
 
 		.avatar-container {
 			display: inline-block;
 			height: $admins-module-icon-size;
 			width: $admins-module-icon-size;
+
+			svg {
```

Comment:
Hmm, I wonder if we can use a similar approach as with  avatar-badge-small mixin?